### PR TITLE
Update LabVIEW measurement examples with the latest measurement template

### PR DESCRIPTION
**Pull request by Jayaseelan-James**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?
This PR updates the 4 LabVIEW measurement examples with the latest measurement template i.e., adding "Measurement Service Helpers" library to the examples. 
Though the `NI-DCPower Source DC Voltage` and `Keysight 34401A DMM Measurement` already have the `Measurement Service Helpers.lvlib` added to it, to accommodate the changes requested in #229, the entire library in those projects were updated.

### Why should this Pull Request be merged?
This PR must be merged to make all the existing LabVIEW measurement examples synchronous to the latest Measurement Service template.

### What testing has been done?
1. The working of `NI-DCPower Source DC Voltage` example is verified using the simulator in NI-MAX. 
1. The working of `CancellableMeasurement` and `UIProgressUpdates` examples were also verified by hosting & running it in InstrumentStudio.

### CODE REVIEW

**Comment by Jayaseelan-James** [comment]
@bkeryan, please help validate the `Keysight 34401A DMM Measurement` example with hardware setup.

File: Source/Example Measurements/CancellableMeasurement/CancellableMeasurement.lvproj
Comment (pbirkhol-ni) [inline Source/Example Measurements/CancellableMeasurement/CancellableMeasurement.lvproj:1]: You should revert changes to this example. Adding a helpers library that isn't used adds unnecessary complication.

**Review by pbirkhol-ni — APPROVED** [review]

File: Source/Example Measurements/CancellableMeasurement/CancellableMeasurement.lvproj
Comment (Jayaseelan-James) [reply Source/Example Measurements/CancellableMeasurement/CancellableMeasurement.lvproj:1]: The reason for adding the Helper library to this example was to make all the examples compliant to the new measurement service template. Else, the user might get confused with the missing `Measurement Service Helpers.lvlib`.

Please share your thoughts.

Comment (pbirkhol-ni) [reply Source/Example Measurements/CancellableMeasurement/CancellableMeasurement.lvproj:1]: I disagree. The template is only a starting point. Once generated, customers should feel free to delete and/or modify any of the generated VIs and we should do the same with our examples. I think the confusion will come from having an entire library that is unused.

I made a similar comment here. https://github.com/ni/measurementlink-labview/pull/230#issuecomment-1609843107

**Review by dixonjoel — APPROVED** [review]

**Comment by bkeryan** [comment]
I tested the 34401A example with InstrumentStudio and TestStand and didn't encounter any problems.

**Pull request merged**